analyzer/src/funTest/kotlin/GoDepTest.kt

```diff
@@ -34,10 +35,13 @@ import io.kotlintest.specs.FreeSpec
 import java.io.File
 
 class GoDepTest : FreeSpec() {
+    private val rootDir = File(".").searchUpwardsForSubdirectory(".git")!!
+    private val projectDir = File(rootDir, "analyzer/src/funTest/assets/projects")
+
     init {
         "GoDep should" - {
             "resolve dependencies from a lockfile correctly" {
-                val projectDir = File("src/funTest/assets/projects/external/qmstr")
+                val projectDir = File("$projectDir/external/qmstr")
```

Comment:
Why did you switch to this `File` constructor from the one that takes a `File` object and a string?

Reply:
I just noticed that the call you replaced also used this constructor. Now I'm confused about when to use which. :)

Reply:
LOL, it's more or less a matter of taste / style, I guess. However, I indeed slightly prefer the version that takes a `File` object to make the base / root directory more stand out. Also, I just realized I'm shadowing the `projectDir` variable (sigh, too much search & replace...), will fix that, too.